lib/struct.js

```diff
@@ -184,6 +186,16 @@ function toString () {
 }
 
 /**
+ * Change struct padding mode.
+ *
+ * @api public
+ */
+function usePack (flag) {
```

Comment:
Let's just make the 2nd argument be an options Object then, with `packed: true` being one of the options.

``` js
var struct = new Struct(spec, { packed: true });
```